### PR TITLE
Fix a typo in the vignette of programming with dplyr

### PR DESCRIPTION
**Pull request by codetrainee**
`quos` and `enquos` are both acceptable here. But I believe `enquos` would be more consistent and intuitive in the context of the vignette.

### CODE REVIEW

File: vignettes/programming.Rmd [hunk 1]
```diff
@@ -432,13 +432,15 @@ variables. We need to make three changes:
 *   Use `...` in the function definition so our function can accept any number
     of arguments.
 
-*   Use `quos()` to capture all the `...` as a list of formulas.
+*   Use `enquos()`/`quos()` to capture all the `...` as a list of formulas.
```
Comment (lionel-) [inline vignettes/programming.Rmd]:
```suggestion
*   Use `enquos()` to capture all the `...` as a list of formulas.
```

File: vignettes/programming.Rmd [hunk 2]
```diff
@@ -432,13 +432,15 @@ variables. We need to make three changes:
...
 
 *   Use `!!!` instead of `!!` to __splice__ the arguments into `group_by()`.
 
 ```{r}
 my_summarise <- function(df, ...) {
   group_var <- enquos(...)
+  # `quos()` can be interchangeably used here.
```
Comment (lionel-) [inline vignettes/programming.Rmd]: Can you remove this comment please?

**Comment by lionel-** [comment]
Thanks!

File: vignettes/programming.Rmd (hunk 2 above)
Comment (codetrainee) [reply vignettes/programming.Rmd]: Thanks. I’ve removed the `quos()` Comments.

**Pull request merged**